agent/config/default.go

```diff
@@ -181,6 +181,11 @@ func DevSource() Source {
 		ports = {
 			grpc = 8502
 		}
+		
+		peering {
+			# We use peer registration for various testing
+			test_allow_peer_registrations = false
+		}
```

Comment:
```suggestion
```

Reply:
To @kisunji point in his other comment, we shouldn't need to have this be disabled here in the DevSource as the NonUserSource will be applied anyways.